internal/events/token_pool_created.go

```diff
@@ -41,21 +41,22 @@ func addPoolDetailsFromPlugin(ffPool *fftypes.TokenPool, pluginPool *tokens.Toke
 	}
 }
 
-func poolTransaction(pool *fftypes.TokenPool, status fftypes.OpStatus) *fftypes.Transaction {
+func poolTransaction(pool *fftypes.TokenPool, status fftypes.OpStatus, blockchainTXID string) *fftypes.Transaction {
 	return &fftypes.Transaction{
-		ID:        pool.TX.ID,
-		Status:    status,
-		Namespace: pool.Namespace,
-		Type:      pool.TX.Type,
+		ID:            pool.TX.ID,
+		Status:        status,
+		Namespace:     pool.Namespace,
+		Type:          pool.TX.Type,
+		BlockchainIDs: fftypes.NewFFStringArray(blockchainTXID),
 	}
 }
 
-func (em *eventManager) confirmPool(ctx context.Context, pool *fftypes.TokenPool, ev *blockchain.Event) error {
+func (em *eventManager) confirmPool(ctx context.Context, pool *fftypes.TokenPool, ev *blockchain.Event, blockchainTXID string) error {
```

Comment:
I think the blockchain ID is now inside the `blockchain.Event`, right? so no need to pass it separately also

Reply:
Sorry I missed this one